apps/web/src/hooks/useTemplates.ts

```diff
@@ -12,7 +12,11 @@ export type INotificationTemplateExtended = INotificationTemplate & {
 };
 
 /** allow override of paginated inputs */
-export function useTemplates(pageIndex?: number, pageSize?: number) {
+export function useTemplates({
+  pageIndex,
+  pageSize,
+  areSearchParamsEnabled = false,
```

Comment:
the flag that is used to control if the query params will be shown in the URL. For example, on the Activity Feed page, we don't want to show it, but the hook is used there for the filter field data.